open-metadata-implementation/governance-servers/data-engine-proxy-services/data-engine-proxy-services-server/src/main/java/org/odpi/openmetadata/governanceservers/dataengineproxy/rest/DataEngineProxyRestService.java

```diff
@@ -17,15 +17,15 @@ public class DataEngineProxyRestService {
     private final DataEngineProxyInstanceHandler instanceHandler = new DataEngineProxyInstanceHandler();
     AtomicBoolean isRunning = new AtomicBoolean(false);
 
-    public ProcessLoadResponse load(String serverName, String userId) {
+    public ProcessLoadResponse initialLoad(String serverName, String userId) {
         String serviceOperationName = "DataEngineProxy";
         try {
             if (isRunning.get()) {
                 return new ProcessLoadResponse();
             }
             DataEngineProxyService dataEngineProxyService = instanceHandler.getDataEngineProxyService(userId, serverName, serviceOperationName);
             isRunning.set(true);
-            CompletableFuture.runAsync(dataEngineProxyService::load).thenAccept(result -> isRunning.set(false));
+            CompletableFuture.runAsync(dataEngineProxyService::initialLoad).thenAccept(result -> isRunning.set(false));
```

Comment:
No applicable, there is error handling/auditing in the underlying service and no error propagation which is fine since this is intended as asynchronous operation call.

Reply:
@sonatype-lift ignore